src/index.js

```diff
@@ -0,0 +1,180 @@
+        lat: location.latitude,
+        lon: location.longitude,
+      },
+    })
+    .then((response) => {
+      let targetTemp = response.data.main.temp
+      return Math.round(convertKtoF(targetTemp));
+    })
+    .catch((error) => {
+      console.log('Error getting the weather:', error);
+    });
+  };
+  
+  const applyColorAndGarden = () => {
+    state.tempValue.textContent = state.tempInt;
+    
+    let landscape = '';
+    if (state.tempInt < 0 || state.tempInt > 120) {
+      state.tempValue.style.color = 'red';
+      state.tempValue.textContent = 'DANGER'
+      landscape = '🚫🚫🚫TOO MUCH WEATHER🚫🚫🚫'; 
+    } else if (state.tempInt>= 80) {
+      state.tempValue.style.color = 'red';
+      landscape = '🌵🏜️🌵🐍🦂🌋👹🌋🦂🐍🌵🏜️🌵';
+    } else if (state.tempInt>= 70) {
+      state.tempValue.style.color = 'orange';
+      landscape = '🌞🌞🌻🌻🏖️🏝️👙🏝️🏖️🌻🌻🌞🌞';
+    } else if (state.tempInt>= 60) {
+      state.tempValue.style.color = 'gold';
+      landscape = '🌹🌸🎋🌹🌸🌱🪺🌱🌸🌹🎋🌸🌹';
+    } else if (state.tempInt>= 50) {
+      state.tempValue.style.color = 'green';
+      landscape = '🌳🌾🍂🪵🌳🐿️🧣🐿️🌳🪵🍂🌾🌳';
+    } else if (state.tempInt>= 40) {
+      state.tempValue.style.color = 'teal';
+      landscape = '🌲🌳🌨️🪵🌲🌳🧤🌳🌲🪵🌨️🌳🌲';
+    } else if (state.tempInt< 40) {
+      state.tempValue.style.color = 'aqua';
+      landscape = '🎄❄️🌨️🌲⛄️🌨️❄️🌨️⛄️🌲🌨️❄️🎄'; 
+    }
+    
+    state.landscapeElement.textContent = landscape;
+  };
+  
+  const increaseTemp = () => {
+    state.tempInt++;
+    applyColorAndGarden();
+  };
+  
+  const decreaseTemp = () => {
+    state.tempInt--;
+    applyColorAndGarden();
+  };
+  
+  const updateCity = () => {
+    state.city = state.cityNameInput.value;
+    state.cityHeader.textContent = state.city;
+  }
+
+  
+  const skyView = (event) => {
+    let skyColor = event.target.value;
+
+  
+    if (skyColor === 'Cloudy') {
+      state.sky.textContent = '☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️';
+      state.gardenSection.setAttribute('class', 'garden__section cloudy')
+      }else if (skyColor === 'Sunny') {
+      state.sky.textContent = '☁️   🌞☀️  🌤️  ☁️ ☀️ ☁️  ☁️';
+      state.gardenSection.setAttribute('class', 'garden__section sunny');
+      }else if (skyColor === 'Rainy') {
+      state.sky.textContent = '🌧🌈☔️🌧🌧💧🌧🌦🌧💧☔️🌧🌧';
+      state.gardenSection.setAttribute('class', 'garden__section rainy')
+      }else if (skyColor === 'Snowy') {
+      state.sky.textContent = '🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨';
+      state.gardenSection.setAttribute('class', 'garden__section snowy');
+    }
+  };
+  
+
+  
+  const resetCity = () => {
+    state.cityNameInput.value = '';
+    state.cityHeader.textContent = '';
+  };
+
+  const handleCurrentTempButtonClicked = () => {
+    findLatAndLon()
+      .then((location) => getWeather(location))
+      .then((targetTemp) => {
+        console.log(targetTemp)
+        state.tempInt = targetTemp;
+        applyColorAndGarden();
+      })
+  }
+  
+  const registerEventHandlers = () => {
+    loadControls();
```

Comment:
By adding the call to `loadControls` inside `registerEventHandlers`, the function is doing more than one thing. Thinking about testing, we'd have to do a bit more set up to test this function since it needs to confirm that both loading and registering happened successfully. I would consider reorganizing this to have a separate function named something like `setUpPage` that first calls `loadControls` then calls `registerEventHandlers`. 

Our situation here is small enough that we may not see the benefit, but consider a larger project where maybe we could pass functions to use into the `registerEventHandlers` function. We'd want to be able to call that independently with different parameters without needing to reload controls that we already have references to.